.travis.yml

```diff
@@ -13,13 +13,15 @@ services:
     - memcached
     - redis-server
 
-before_script:
+before_install:
     - sh -c "if [ $TRAVIS_PHP_VERSION != 'hhvm' ]; then pecl install riak-beta; fi"
     - sh -c "if [[ $TRAVIS_PHP_VERSION != 'hhvm' && `php-config --vernum` -ge 50500 ]] ; then pecl config-set preferred_state beta; printf "yes\n" | pecl install apcu ; else echo 'extension="apc.so"' >> ./tests/travis/php.ini ;fi"
     - composer self-update
-    - composer --prefer-source --dev install
     - sh -c "if [ $TRAVIS_PHP_VERSION != 'hhvm' ]; then phpenv config-add ./tests/travis/php.ini; fi"
```

Comment:
@stof Sorry, I missed your response. Can you let me know why you want it all in `before_script` or which line exactly? I will try to fix it up ASAP.

cc @Ocramius